prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -3,7 +3,7 @@ package gov.cdc.prime.router
 import com.github.javafaker.Faker
 import com.github.javafaker.Name
 import java.text.SimpleDateFormat
-import java.util.Random
+import java.util.*
 import java.util.concurrent.TimeUnit
```

Comment:
Our rules don't allow for wildcard imports. You should set up Intellij and ktlint to prevent this from happening.

Reply:
Ok.  I was letting IntelliJ take care of all that.